docs/project-scope.md

```diff
@@ -0,0 +1,161 @@
+# Voyager T800 – Project Scope
+
+A generative AI-powered multimodal travel planning assistant developed during a 12-week Data Science internship.
+
+---
+
+## 1. Project Vision
+
+**Goal**:  
+To build an AI assistant that generates personalized, fact-grounded travel itineraries using both **text** and **image** inputs, powered by LLMs and enhanced with retrieval-augmented generation (RAG), agentic reasoning, and vision-based tools.
+
+---
+
+## 2. Target Users
+
+The primary users of the application:
+
+- [ ] Families planning vacations
+- [ ] Budget-conscious solo travelers
+- [ ] Photographers / travel bloggers
+- [ ] First-time city explorers
+- [ ] Adventure seekers / active travelers
+- [ ] International travelers
+- [ ] Couples planning romantic getaways
+- [ ] Locals exploring nearby destinations
+- [ ] Students on cultural trips
+- [ ] Other: `__________`
+
+---
+
+## 3. Key Use Case Scenarios
+
+| **Scenario**                                                | **Input Type**   | **Expected Output**                                                                 |
+| ----------------------------------------------------------- | ---------------- | ----------------------------------------------------------------------------------- |
+| “Plan a 5-day trip to Paris for a foodie couple”            | Text             | Day-by-day itinerary with food stops                                                |
+| "I want to spend a week exploring Kyiv and Lviv"            | Text             | Day-by-day itinerary with main attractions and travel route between cities included |
+| Upload a beach photo + "I want something similar in Europe" | Image + Text     | Destination guess + itinerary with beach options                                    |
+| “Make Day 2 more relaxing”                                  | Follow-up Prompt | Adjusted itinerary                                                                  |
+| Upload a blurry or unrelated photo + "I want to go here"    | Image + Text     | Generic popular destinations itinerary or last saved location itinerary             |
+| Empty or nonsensical input (e.g., “asdfghjkl”)              | Text             | Default help itinerary with sample queries                                          |
+| “Make Day 5 more exciting” when itinerary only has 3 days   | Follow-up Prompt | Revised 3-day itinerary with more activities on Day 3                               |
+
+---
+
+## 4. Core Features (MVP Milestones)
+
+- [ ] Text-to-itinerary generation with LLM
+- [ ] Prompt templates (simple & structured)
+- [ ] Basic Streamlit UI (text input + output)
+- [ ] LangChain chain for itinerary generation
+- [ ] Knowledge base (3 cities max)
+- [ ] Basic RAG with manual retrieval
+- [ ] Multimodal input (image upload)
+- [ ] Vision model integration (tags/captions)
+- [ ] Agent prototype for follow-ups or tools
+- [ ] JSON-structured output
+- [ ] Conversation History
```

Comment:
For each feature, consider linking to a relevant issue or milestone if it exists.
Clarify what “Basic RAG with manual retrieval” means for MVP (e.g., “manual selection of city guides by user prompt”).